packages/js-runtime/src/runtime/fetch.ts

```diff
@@ -8,17 +8,18 @@ export class Headers {
     if (init) {
       if (Array.isArray(init)) {
         init.forEach(([key, value]) => {
-          this.addValue(key, value);
+          this.addValue(key.toLowerCase(), value);
         });
       } else {
         Object.entries(init).forEach(([key, value]) => {
-          this.addValue(key, value);
+          this.addValue(key.toLowerCase(), value);
```

Comment:
We can revert both of these lines, since we already call `toLowerCase()` inside `addValue()`

Reply:
Of course, d'oh!